### PR TITLE
Design layout for homepage

### PR DESCRIPTION
**Pull request by jonathastavares**
This is the first milestone for my JavaScript Capstone project, which is a website related to Video games reviews, rating, descriptions and etc.

Built using:
- HTML
- CSS
- JavaScript
- Webpack
- BootStrap

In this feature, I added a simple design for homepage, displaying 50 games per page.

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ import fetch from 'cross-fetch';
 import './style.css';
 
 const key = '4367d242d87843ddb5e0a8cc46a359d5';
-const url = `https://api.rawg.io/api/games?key=${key}`;
+const quantity = 50;
+const url = `https://api.rawg.io/api/games?key=${key}&page_size=${quantity}`;
 // const gameUrl = `https://api.rawg.io/api/games/${id}`;
```
Comment (shadabwahidullah) [inline src/api.js]: To make your code clean, It's a good practice to get rid of used commented codes.

File: src/index.html [hunk 2]
```diff
@@ -13,29 +13,24 @@
 </head>
 <body >
   <div class="wrapper">
-    <nav class="navbar navbar-expand-lg navbar-light bg-light">
-        <div class="container-fluid">
+    <nav class="navbar navbar-expand-lg navbar-light bg-light opacity-50">
+        <div class="container-fluid mx-5">
           <div class='nav-logo me-3'></div>
```
Comment (shadabwahidullah) [inline src/index.html]: It's good to add a link for your logo, when clicked, to return users to the main page.

File: src/index.html [hunk 3]
```diff
@@ -13,29 +13,24 @@
...
               </li>
             </ul>
+            <p id='counter'>Available Games: </p>
           </div>
         </div>
     </nav>
-    <section id='game-list' class=''>
-      <h2 class='fw-bold fs-1 ms-5 my-4'>Game List</h2>
+    <section>
+      <h2 class='my-2 text-center title'>Game List</h2>
```
Comment (shadabwahidullah) [inline src/index.html:30]: I really like the font for GAME LIST it's cool :+1:

File: src/style.css [hunk 4]
```diff
@@ -25,5 +41,26 @@ body {
...
+}
+
+.btn:hover {
+  background-color: rgb(235, 200, 158);
+  color: rgb(32, 20, 6);
+}
+
+.card-img-top {
+  min-height: 200px;
+  max-height: 200px;
```
Comment (shadabwahidullah) [inline src/style.css]: This is a question for adding to my information. instead of these two, Why haven't you used `height: 200px`?

**Review by shadabwahidullah — CHANGES_REQUESTED** [review]
## :+1: You have done a great job so far
I did like the look and feel of the Homepage

## :recycle: There are some points that I think, It would be better to work on.

- [x] It would be better if you use try and catch incase the could throws an error 
https://github.com/jonathastavares/JavaScript-Capstone/blob/098131e5331487b9430dd13dee03271fc3c33a67/src/api.js#L43-L47

File: src/style.css (hunk 4 above)
Comment (jonathastavares) [reply src/style.css]: I just used it wrong by the way, I set first a min-height, and then I had problems with max-height and I set it after it, and I forgot to DRY the code after that, I just fix that aswell

**Review by shadabwahidullah — APPROVED** [review]
## Approved :clinking_glasses: :partying_face: 
You have done a great job
Wish you luck with other tasks.

**Pull request merged**